Delicious/Scenes/Home/HomeViewController.swift

```diff
@@ -72,8 +76,41 @@ final class HomeViewController: UIViewController, BindableType {
     }
 
     func bindViewModel() {
-        let input = HomeViewModel.Input()
+        let input = HomeViewModel.Input(
+            loadTrigger: loadTrigger.startWith(()).asDriverOnErrorJustComplete(),
+            reloadTrigger: tableView.loadMoreTopTrigger,
+            selectTrigger: selectTrigger.asDriverOnErrorJustComplete().throttle(Constant.throttle),
+            selectItemTrigger: tableView.rx.modelSelected(HomeTableViewItem.self).asDriver().throttle(Constant.throttle),
+            searchTrigger: searchButton.rx.tap.asDriver().throttle(Constant.throttle)
+        )
+        
         let output = viewModel.transform(input)
+        
+        output.isLoading
+            .drive(rx.isLoading)
+            .disposed(by: rx.disposeBag)
+        output.isReloading
+            .drive(tableView.isLoadingMoreTop)
+            .disposed(by: rx.disposeBag)
+        output.tableViewState
+            .drive(tableView.rx.state)
+            .disposed(by: rx.disposeBag)
+        output.loadedError
+            .drive(rx.error)
+            .disposed(by: rx.disposeBag)
+        output.selected
+            .drive()
+            .disposed(by: rx.disposeBag)
+        output.search
+            .drive()
+            .disposed(by: rx.disposeBag)
+        output.data
+            .do(onNext: { _ in self.featuredRecipes.removeAll() })
```

Comment:
```suggestion
            .do(onNext: self.featuredRecipes.removeAll)
```
side effect nhận đầu vào là 1 function, nên em có thể truyền thằng function đó vào luôn

Reply:
Sửa như trên bị báo lỗi vì data ko phải Driver<Void>